docs/assets/overview.md

```diff
@@ -34,8 +42,33 @@ rx.image(src="/Reflex.svg", width="5em")
 # Always prefix the asset path with a forward slash `/` to reference the asset from the root of the project, or it may not display correctly on non-root pages.
 ```
 
-## Favicon
+## Frontend vs Backend Assets
+
+### Current Behavior
+Assets in the `assets/` folder are primarily intended for frontend use. When you reference an asset, it is:
+1. Copied to the frontend build directory during compilation
+2. Served statically by the frontend server
+
+### Using Assets in Backend Code
+While it's possible to access assets from backend code (they are copied to the backend), this is not the recommended approach and may not be supported in future versions. Instead:
+
+1. For backend-only files: Store them outside the `assets/` directory
+2. For files needed in both frontend and backend:
+   - Store a copy in both locations
+   - Use appropriate paths for each context
 
-The favicon is the small icon that appears in the browser tab.
+## Experimental Features Status
+
+The experimental assets module (`rx._x.asset`) is being deprecated in favor of the core implementation. Use `rx.asset` with `shared=True` for all new code:
+
+```python
+# Old way (deprecated)
+rx._x.asset("my_file.png")
+
+# New way
+rx.asset("my_file.png", shared=True)
```

Comment:
`shared=True` was only intended for libraries which want to expose the asset to downstream projects and should not be the default recommendation